.github/workflows/lint.yml

```
@@ -116,3 +116,9 @@ jobs:
     - name: lint remain
       run: |
         make lint-remain
+
+    - name: install markdownlint cli
+      run: npm install -g markdownlint-cli
+
+    - name: lint markdown
+      run: markdownlint '**/*.md' --disable MD012 MD013 MD029 MD033 MD034 MD036 MD041
```

```suggestion
      run: make lint-markdown
```

To avoid repeating yourself.